Description: Detect mips CPUs in ./configure
 This basically recreates the logic that already exists in bootstrap.py
 Endianness test from https://stackoverflow.com/questions/26859098/testing-endianness-of-system-with-the-unix-shell
Author: Ximin Luo <infinity0@debian.org>
Bug: https://github.com/rust-lang/rust/pull/38650
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/test/run-pass/conditional-compile-arch.rs
+++ b/src/test/run-pass/conditional-compile-arch.rs
@@ -28,6 +28,12 @@
 #[cfg(target_arch = "mips64")]
 pub fn main() { }
 
+#[cfg(target_arch = "mips")]
+pub fn main() { }
+
+#[cfg(target_arch = "mips64")]
+pub fn main() { }
+
 #[cfg(target_arch = "powerpc64")]
 pub fn main() { }
 
